week5_R_draft.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
<p>typo in second bullet</p><p> </p><p>make clearer what are headerr and rownames in your dataframe (so they don't confuse obs with a column)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks for catching that!</p>